### DISCUSSION TITLE
GitHub Renovate App - Cross Organization Access?

### DISCUSSION THREAD

**Discussion by ekristen** [renovatebot/renovate#11797] [Request Help] [answered]
I have a GitHub Enterprise with multiple organizations but most of the repos are private, we are using the GitHub App Renovate Bot version.

When renovate runs in OrgA, it gets 404 when trying to resolve dependency information from repo in OrgB.

Is there any way to grant it access to read repos in OrgA?

**Comment by rarkins** [comment]
The GitHub app can only generate tokens scoped to a single org. If you configure a host rule using matchHost=https://github.com/theotherorg/ and a personal access token then it might work (might also need hostType configured depending on what you're using it for)

**Reply by ekristen** [reply]
Here are the logs ...

```
DEBUG: GitHub 404
{
  "url": "https://api.github.com/repos/orgB/repo1/releases?per_page=100"
}
DEBUG: Datasource 404
{
  "datasource": "github-releases",
  "lookupName": "orgB/repo1",
  "url": "https://api.github.com/repos/orgB/repo1/releases?per_page=100"
}
```

will doing a matchHost like you've described work even though it's hitting the api? 

do I just then add a `token` field and encrypt it with the renovate bot encrypt secret website?

**Reply by rarkins** [reply]
matchHost=https://api.github.com/repos/orgB/
hostType=github-releases
encrypted.token=......

**Reply by ekristen** [reply]
Copy, will give that a try asap. I appreciate it.

**Reply by ekristen** [reply]
For those who follow, there is the working repository configuration.

```json
{
    "hostRules": [
        {
            "matchHost": "https://api.github.com/repos/organizationA/",
            "hostType": "github-releases",
            "encrypted": {
                "token": "wcFMA..."
            }
        }
    ]
}
```

**Comment by ekristen** [comment, marked as answer]
**Note:** for anyone arriving here, this no longer works due to renovate now using the graphql interface. Discussions are in the works on how to find a new approach to allow this to work.

Thanks to @rarkins, here is the working configuration.

```json
{
    "hostRules": [
        {
            "matchHost": "https://api.github.com/repos/organizationA/",
            "hostType": "github-releases",
            "encrypted": {
                "token": "wcFMA..."
            }
        }
    ]
}
```

**Reply by ekristen** [reply]
This has stopped working, opened up a  new app support discussion https://github.com/renovatebot/renovate/discussions/15978
